### ISSUE TITLE
User is not navigated to the activity tab after a bridge transaction is performed

### ISSUE THREAD

**Issue by VolodLytvynenko** [status-im/status-react#20534]
Unfortunately, I was not able to report this issue initially within https://github.com/status-im/status-mobile/pull/19984 due to a bridge issue. I decided to postpone and retest it when bridge routes are able to be found.

### Steps:
1. Recover a user with available assets on mainnet (only on mainnet mode can the bridge be built).
2. Go to the bridge flow.
3. Select the network assets you want to bridge (Note: The mainnet bridge can be expensive, costing 30-70 USD. It is recommended to build the L2 bridge only, such as Opt -> Arb or Arb -> Opt where fee is less).
4. Perform the transaction.

### Actual result:
- After the transaction is performed, the user is navigated to the list of assets on the bridge page.
- "invalid argument 2:json: cannot unmarshal non-string" error is shown

![image](https://github.com/status-im/status-mobile/assets/52490791/117f9e3f-b2f6-4c0b-b7fe-f445072201bf)

### Expected result:
After the transaction is performed, the user should be navigated to the activity page of the account from which the transaction originated.

### Logs:
[logs.zip](https://github.com/user-attachments/files/15928483/logs.zip)

### ENV:
NIghtly 21 Jun 2024

**Comment by churik** [comment]
blocked by non-working bridge https://github.com/status-im/status-mobile/pull/20631#issuecomment-2252513966
